version_history.txt

```diff
@@ -1,4 +1,5 @@
 v0.6.5
+- do not allow for meshes named after bones
```

Comment:
this could use a better description

Reply:
do not allow a mesh and bone to share the same name?